src/sidebar/store/modules/annotations.js

```diff
@@ -281,6 +290,55 @@ function addAnnotations(annotations) {
   };
 }
 
+/** Set the currently displayed annotations to the empty set. */
+function clearAnnotations() {
+  return { type: actions.CLEAR_ANNOTATIONS };
+}
+
+/**
+ * Replace the current set of focused annotations with the annotations
+ * identified by `tags`. All provided annotations (`tags`) will be set to
+ * `true` in the `focused` map.
```

Comment:
The existence of the `focused` map is effectively an implementation detail since outside code will now only access the store through its selectors. Same applied to the highlight action creators below.

Reply:
What you have said here is true :). What changes would you like to see in the code/comments?